tools/serverpod_cli/lib/src/generator/dart/library_generators/test_tools_generator.dart

```diff
@@ -0,0 +1,400 @@
+import 'package:code_builder/code_builder.dart';
+import 'package:path/path.dart' as p;
+import 'package:serverpod_cli/analyzer.dart';
+import 'package:serverpod_cli/src/analyzer/dart/definitions.dart';
+import 'package:serverpod_cli/src/generator/shared.dart';
+
+class TestToolsGenerator {
```

Comment:
Suggestion: Rename to `ServerTestToolsGenerator`.

Since this is not a general test tools generator and we later on might want some kind of "mock" that we generate.